src/objective-c/tests/run_one_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Don't run this script standalone. Instead, run from the repository root:
+# ./tools/run_tests/run_tests.py -l objc
+
+set -ev
+
+cd $(dirname $0)
+
+BINDIR=../../../bins/$CONFIG
+
+[ -f $BINDIR/interop_server ] || {
+    echo >&2 "Can't find the test server. Make sure run_tests.py is making" \
+             "interop_server before calling this script."
+    exit 1
+}
+$BINDIR/interop_server --port=5050 --max_send_message_size=8388608 &
```

Comment:
exposing interop_server on fixed ports assumes that objc are the only tests running on the test machine at the moment. That currently is true (because of the `-j 1` argument here https://github.com/grpc/grpc/blob/4dd3afbad9acd09a5b24b312e5820bb82f22b1e1/tools/internal_ci/macos/grpc_basictests_objc_dbg.cfg#L30), but it's a very fragile assumption.

Fixed ports make it easy for the tests to collide with each other when not careful. 
Also ObjC is the only language that relies on fixed ports for running tests.

Reply:
That sounds reasonable. I'll make it use port server.